uninstall

```diff
@@ -86,21 +87,21 @@ else
     echo -e "\nNo stor directory found, skipping..."
 fi
 
-echo -e "\nUninstalling Airtime..."
-service airtime-celery stop
+echo -e "\nUninstalling LibreTime..."
+systemctl stop libretime-celery
 set +e
-removeRabbitmqAirtimeSettings
+removeRabbitmqLibreTimeSettings
 set -e
 
 for i in ${FILES[*]}; do
     rm -rf $i
 done
 
-echo -e "\nDo you want to drop your current Airtime database? [y/N]: \c"
+echo -e "\nDo you want to drop your current LibreTime database? [y/N]: \c"
 read IN
 if [[ "$IN" = "y" || "$IN" = "Y" ]]; then
-    echo -e "\nDropping Airtime database..."
-    dropAirtimeDatabase
+    echo -e "\nDropping LibreTime database..."
+    dropLibreTimeDatabase
 fi
 
 pip3 uninstall -y airtime-playout airtime-media-monitor airtime-analyzer
```

Comment:
Do we not have the packages renamed in pip?

Reply:
No, not yet